lib/ASRouter.jsm

```diff
@@ -2081,9 +2172,11 @@ class _ASRouter {
         QueryCache.expireAll();
         break;
       case "ENABLE_PROVIDER":
+        await this.unblockProviderById(action.data);
```

Comment:
I added these in in order to have a way to clear provider impressions through ASRouter admin. It would not cover the cases where there are impressions from a group that is different from an existing provider though.